src/appMain/signal_handlers.cc

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
@pestOO, this functionality is redundant, we process system signals in utils/src/signals_win (utils/src/signals_posix)